src/reactor.cr

```diff
@@ -145,6 +149,22 @@ module Mint
         SourceFiles.external_stylesheets.to_s
       end
 
+      get "/assets/:name" do |env|
```

Comment:
Shouldn't this use rather the asset path?

```suggestion
      get "/assets/*path" do |env|
```

Reply:
No the assets cannot be in nested paths so `:name` is sufficient, but it should use the `ASSET_DIR` constant.